app/views/tournaments/edit.html.erb

```diff
@@ -78,8 +80,6 @@
           <div class="tourney-form"><%= f.input :name %></div>
           <div class="tourney-form"><%= f.input :rating %></div>
           <div class="tourney-form"><%= f.input :division %></div>
-          <div class="tourney-form"><%= f.input :seed %></div>
```

Comment:
You might want to update the players controller to automatically assign them a seed and ranking (based on the lowest seed/ranking + 1)

Reply:
Even though AFAIK we don't use those variables after the start of round 1